widget/ui/src/components/Toast/Toast.styles.ts

```diff
@@ -0,0 +1,113 @@
+import { darkTheme, keyframes, lightTheme, styled } from '../../theme';
+import { Typography } from '../Typography';
+
+const toastOpacity = keyframes({
+  from: { opacity: 0 },
+  to: { opacity: 1 },
+});
+
+export const ToastContentContainer = styled('div', {
+  variants: {
+    horizontal: {
+      right: {
+        right: 12,
+        animation: `${toastOpacity} .7s`,
+      },
+      left: {
+        left: 12,
+        animation: `${toastOpacity} .7s`,
+      },
+    },
+  },
+});
+
+export const ToastContainer = styled('div', {
+  position: 'absolute',
+  zIndex: 9999,
+  display: 'flex',
+  gap: '$4',
+  flexDirection: 'column',
+  variants: {
+    horizontal: {
+      right: {
+        right: 12,
+      },
+      left: {
+        left: 12,
+      },
+    },
+
+    vertical: {
+      top: {
+        top: 2,
+      },
+      bottom: {
+        bottom: 2,
+      },
+    },
+  },
+});
+
+export const AlertBox = styled('div', {
+  display: 'flex',
+  borderRadius: '$sm',
+  width: '292px',
+  minHeight: '52px',
+  backgroundColor: '$background',
+  borderRight: '1px solid $foreground',
+  [`.${darkTheme} &`]: {
+    backgroundColor: '#070917',
+  },
+  '&:hover': {
+    backgroundColor: '$info100',
+    [`.${darkTheme} &`]: {
+      backgroundColor: '#111733',
+    },
+  },
+  variants: {
+    type: {
+      error: {
+        borderRight: '1px solid $error500',
```

Comment:
`borderRightColor` is enough for all variants. it helps to change the border width from a single place (line 57).

Reply:
fixed.